.changeset/lemon-bees-confess.md

```diff
@@ -1,5 +1,8 @@
 ---
-'@shopify/polaris': minor
+'@shopify/polaris': major
+'polaris.shopify.com': major
 ---
 
-Migrated `space` custom properties from v11 to v12
+- Migrated `space` custom properties from v11 to v12
+- Removed backwards compatibility for v11 space tokens on `Tooltip` component `Padding` type
+- Updated JSDoc prop type descriptions to include updated `border` custom properties
```

Comment:
Since I updated the backwards compatibility of `Padding` type on `Tooltip` (after have convo about removing backwards compatibility in `next` while running migrations after already merging space migration), I snuck in a change to update the changeset for space here.